src/Writer/AbstractFeed.php

```diff
@@ -151,11 +154,14 @@ public function setDateCreated($date = null)
     /**
      * Set the feed modification date
      *
-     * @param  null|int|DateTimeInterface
-     * @return $this
+     * @param null|int|DateTimeInterface
+     * @param DateTime|\DateTimeImmutable|int|null|string $date
+     *
+     * @return self
+     *
      * @throws Exception\InvalidArgumentException
      */
-    public function setDateModified($date = null)
+    public function setDateModified($date = null): self
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.